generators/dockertools/templates/java/cli-config.yml.template

```diff
@@ -27,13 +27,13 @@ image-name-tools : "bx-dev-java-maven-tools"
 host-path-run : "target"
 
 # The command to build the code and docker image for RUN
-build-cmd-run : "mvn install -Dmaven.repo.local=/project/.m2/repository"
+build-cmd-run : "mvn install -Dmaven.repo.local=/project/.m2/repository -DoutputDirectory=/tmp/liberty"
 
 # The command to execute tests for the code in the tools container
-test-cmd : "mvn install -Dmaven.repo.local=/project/.m2/repository"
+test-cmd : "mvn install -Dmaven.repo.local=/project/.m2/repository -DoutputDirectory=/tmp/liberty"
 
 # The command to build the code and docker image for DEBUG
-build-cmd-debug : "mvn install -Dmaven.repo.local=/project/.m2/repository"
+build-cmd-debug : "mvn install -Dmaven.repo.local=/project/.m2/repository -DoutputDirectory=/tmp/liberty"
 {{/has}}
```

Comment:
Does this change need to be done for Gradle as well?

Reply:
I would like to fix the maven first as we need to change the build.gradle file to accept the alternate server output directory from a -P property from the gradle command.

The branch should be up to date now.